content/guides/models/registry/search_registry.md

```diff
@@ -7,13 +7,89 @@ title: Find registry items
 weight: 7
 --- 
 
-Use the global search bar in the W&B Registry App to find a specific registry, collection, artifact version tag, collection tag, or alias. To search globally:
+Use the global search bar in the W&B Registry App or Python SDK to find a specific registry, collection, collection tag, artifact version name, artifact tag, or artifact alias.
 
-1. Navigate to the W&B Registry App at https://wandb.ai/registry/. 
+Only items that you have permission to view appear in the search results.
+
+## Interactively search for registry items
```

Comment:
I don't love having a split infinitive in the heading. Can we get rid of the adverb?